.azure-pipelines.yml

```diff
@@ -122,6 +147,11 @@ jobs:
 
 - job: ACC_1804_SGX_perf_tests
   pool: Ubuntu-1804-DC4s
+  container:
+    image: ccfciteam/ccf-ci-18.04-oe-0.6-sgx:latest
+    # --publish-all enables end-to-end tests to communicate over ports
+    # --device /dev/sgx:/dev/sgx makes sgx available to the container
+    options: --publish-all --device /dev/sgx:/dev/sgx -v /mnt/build:/__w/
```

Comment:
The issue with the publishing tasks inside a container has been reported here: https://github.com/microsoft/azure-pipelines-tasks/issues/11137

We should be able to remove the mount once it is solved.